File: ptt-gc-spa/src/app/core/models/customReportHeader.ts
export interface CustomReportHeader {
    reportID: number;
    reportCode: string;
    reportName: string;
    description: string;
    graphType: string;
    x_AxisLabel: string;
    y_AxisLabel: string;
    otherTypeLabel: string;
    createUser: string;
    createDate: Date;
    updateUser: string;
    updateDate: Date;
    stageType: Date;
    isAccumulate: boolean;
}

File: ptt-gc-api/Models/SettingChart/CustomReportHeader.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace PTT_GC_API.Models.SettingChart
{
    public class CustomReportHeader
    {       
        [Key]
        public int RunningID { get; set; }
        public int? ReportID { get; set; }
        public string ReportCode { get; set; }
        public string ReportName { get; set; }
        public string Description { get; set; }
        public string GraphType { get; set; }
        public string X_AxisLabel { get; set; }
        public string Y_AxisLabel { get; set; }
        public string OtherTypeLabel { get; set; }
        public string CreateUser { get; set; }
        public DateTime? CreateDate { get; set; }
        public string UpdateUser { get; set; }
        public DateTime? UpdateDate { get; set; }
        public string StageType { get; set; }
        public Boolean? isAccumulate { get; set; }
        public string SystemReportType { get; set; }

    }
}
